contrib/build-meep.sh

```diff
@@ -63,6 +72,28 @@ Press return to continue
 EOF
 read junk
 
+if $ubuntu; then
+    RPATH_FLAGS="-Wl,-rpath,${DESTDIR}/lib:/usr/lib/x86_64-linux-gnu/hdf5/openmpi"
+    LDFLAGS="-L${DESTDIR}/lib -L/usr/lib/x86_64-linux-gnu/hdf5/openmpi ${RPATH_FLAGS}"
+    CFLAGS="-I${DESTDIR}/include -I/usr/include/hdf5/openmpi"
+fi
+
+if $centos; then
+    export CC=/usr/lib64/openmpi/bin/mpicc
+    export CXX=/usr/lib64/openmpi/bin/mpicxx
+    export PATH=${PATH}:/usr/lib64/openmpi/bin
+    RPATH_FLAGS="-Wl,-rpath,${DESTDIR}/lib64:/usr/lib64/openmpi/lib"
+    LDFLAGS="-L${DESTDIR}/lib64 -L/usr/lib64/openmpi/lib ${RPATH_FLAGS}"
+    CFLAGS="-I${DESTDIR}/include -I/usr/include/openmpi-x86_64/"
+    export PYTHONPATH=${DESTDIR}/lib/python3.6/site-packages
+    export PYTHONPATH=${PYTHONPATH}:/usr/local/lib64/python3.6/site-packages
+    export PYTHONPATH=${PYTHONPATH}:/usr/local/lib64/python3.6/site-packages/mpi4py
+    export LD_LIBRARY_PATH=${DESTDIR}/lib64:/usr/lib64/openmpi/lib:/usr/local/lib64/python3.6/site-packages/mpi4py/lib-pmpi:/usr/local/lib64/python3.6/site-packages/mpi4py
```

Comment:
Doesn't `mpicc` take care of the library path for us?

Reply:
Yes, removed in next commit.